<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-H0GN5MM" failures="1" tests="1" name="Default test(failed)(failed)(failed)" time="0.091" errors="0" timestamp="01 Mar 2019 05:37:35 GMT">
  <testcase classname="demo.TestNGFailedTC" name="Test3" time="0.027">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:55)
at junit.framework.Assert.assertTrue(Assert.java:22)
at junit.framework.Assert.assertTrue(Assert.java:31)
at demo.TestNGFailedTC.Test3(TestNGFailedTC.java:27)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- Test3 -->
  <testcase classname="demo.TestNGFailedTC" name="Test2" time="0.0">
    <ignored/>
  </testcase> <!-- Test2 -->
  <testcase classname="demo.TestNGFailedTC" name="Test1" time="0.0">
    <ignored/>
  </testcase> <!-- Test1 -->
</testsuite> <!-- Default test(failed)(failed)(failed) -->
